infracost/data_source_aws_apigatewayv2_api.go

```diff
@@ -0,0 +1,18 @@
+package infracost
+
+import (
+    "github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsApiGatewayV2Api() *schema.Resource {
+    return &schema.Resource{
+        Read: dataSourceRead,
+        Schema: map[string]*schema.Schema{
+            "resources":                resourcesSchema(),
+            "average_message_size":     usageSchema(),
```

Comment:
I wonder if we should change usageSchema to accept an optional validateFunc? see https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#validatefunc 

@aliscott any thoughts?

Reply:
Makes sense to me

Reply:
Thanks @aliscott, I created https://github.com/infracost/terraform-provider-infracost/issues/13 so we can don't block this PR from being merged.